fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -4710,14 +4710,9 @@ void clearIndexData(@Nonnull Index index) {
         IndexingRangeSet.forIndexBuild(this, index).clear();
         // clear even if non-unique in case the index was previously unique
         context.clear(indexUniquenessViolationsSubspace(index).range());
-        // Under the index build subspace, there are 3 lower level subspaces, the lock space, the scanned records
```

Comment:
This comment about why we aren't erasing the log seems valuable, and should be preserved, probably on the javadoc for `eraseAllIndexingDataButTheLock`